pkg/tasks/c1api/actions.go

```diff
@@ -69,19 +75,23 @@ func (c *actionGetSchemaTaskHandler) HandleTask(ctx context.Context) error {
 
 	t := c.task.GetActionGetSchema()
 	if t == nil || t.GetName() == "" {
-		return c.helpers.FinishTask(ctx, nil, nil, errors.New("action name required"))
+		return c.helpers.FinishTask(ctx, nil, t.GetAnnotations(), errors.New("action name required"))
 	}
 
 	resp, err := cc.GetActionSchema(ctx, &v2.GetActionSchemaRequest{
-		Name: t.GetName(),
+		Name:        t.GetName(),
+		Annotations: t.GetAnnotations(),
 	})
 	if err != nil {
-		return c.helpers.FinishTask(ctx, nil, nil, err)
+		return c.helpers.FinishTask(ctx, nil, t.GetAnnotations(), err)
```

Comment:
I think the thing returned should always be resp.Annotations so we don't mix them up.